### ISSUE TITLE
DST Staging Review: (Consider) Heading hierarchy may not be clear when status tags are used

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#5541]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** va-tag-status
**Staging Review ticket:** [Staging Review: va-tag-status component](https://github.com/department-of-veterans-affairs/va.gov-team/issues/129077)

## Findings details
**VA.gov Experience Standard - issue:** [User has enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**This is an issue with the:** 
**High-priority:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

Because our card component does not have any semantic structure, incudling the status tag above the card heading, may be confusing and assistive technoloigy users, specifically screeen reader users, may not understand that the Status tag is attached to the heading below it and may believe it is attached to the heading above it. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="309" height="389" alt="stacked cards" src="https://github.com/user-attachments/assets/1ebe81b1-ded9-4d36-b33e-51dbff62c286" />

</details>

## Recommended action
Note: this issue is dependent on the following question: Can there be stacked cards with a status tag that do not have a heading between them?

If so, consider the screen reader experience, and either provide semantic structure to the cards, so that that structure helps build an understanding that the status tag and heading are part of the same structure, or determine another approach to build that understanding. 

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG22/#info-and-relationships)
- **Modality:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** [Design Foundation - Overview](https://design.va.gov/foundation/)
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for DST

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation
